pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,45 @@
+{ mkDerivation
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  qp = "./src/utils/qpaeq";
+in
+mkDerivation rec {
+  pname = "qpaeq";
+  inherit (pulseaudio) version src;
+
+  buildInputs = [
+    (
+      (python3.withPackages(ps: with ps; [ pyqt5 dbus-python sip ]))
```

Comment:
Does not `pyqt5` already propagate compatible `sip`?

Reply:
It doesn't add the required metadata so it can't find the sip package, adding both fixes that

Reply:
If that's ok resolve conversation, otherwise comment

Reply:
How do other `pyqt5` packages solve this issue?

Reply:
I'm seeing lots of --pyqt-sipdir

```
./development/python-modules/qscintilla-qt5/default.nix:39:      --pyqt-sipdir=${pyqt5}/share/sip/PyQt5 \

./development/python-modules/poppler-qt5/default.nix:26:    "--pyqt-sip-dir ${pyqt5}/share/sip/PyQt5"
```

Seems that they overwrite preConfigure

Reply:
Note that most of those are python modules that get built, not just some random standalone files. So unsure how to proceed.

Reply:
What error message does it produce when you omit sip?

Reply:
```
There was an error importing needed libraries
Make sure you have qt5 and dbus-python installed
The error that occurred was:
	No module named 'sip'
```

Reply:
Does using pyqt5.sip fix the conflict?

Reply:
Fixed